runtime.md

```diff
@@ -135,6 +135,11 @@ This operation MUST [generate an error](#errors) if it is not provided the conta
 Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
 This operation MUST send the specified signal to the container process.
 
+### <a name="runtimeKillAll" />KillAll
+`kill <container-id> <signal> <all>`
```

Comment:
```markdown
### <a name="runtimeKill" />Kill
`kill <container-id> <signal> <all>`
This operation MUST [generate an error](#errors) if it is not provided the container ID.
Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
This operation MUST send the specified signal to the container process.

If `<all>` is `true`, all the processes in the container are killed.
```

Or:

```markdown
### <a name="runtimeKill" />Kill
`kill <container-id> <signal>
This operation MUST [generate an error](#errors) if it is not provided the container ID.
Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect on the container and MUST [generate an error](#errors).
This operation MUST send the specified signal to the container process.

### <a name="runtimeKillAll" />KillAll
`kill-all <container-id> <signal>`

Kill all the processes in the container.
```

Reply:
I have picked up the first one. Thanks 😍